### DISCUSSION TITLE
Trouble connecting minestom-ce 1.20.1 server with Velocity

### DISCUSSION THREAD

**Discussion by hugovdev** [Minestom/Minestom#1966] [Help] [answered]
I connected a minestom server to a Velocity server and I can't seem to connect to the Minestom server.
I enabled the VelocityProxy in Minestom with the secret as well
![image](https://github.com/Minestom/Minestom/assets/23460704/bbf640a4-1183-40f9-8a6f-f4d343d4ec39)

Player seems to connect and disconnect, connect does reach the lobby server.
![image](https://github.com/Minestom/Minestom/assets/23460704/d32d771a-e4a2-44c1-a9a9-bff9a8d5e9e6)

**Comment by mworzala** [comment, marked as answer]
There appears to be an incompatibility with ViaVersion when connecting with a 1.20.2 client joining a 1.20.1 Minestom server. ViaVersion does not officially support Minestom and has not been helpful in debugging this. For now it will remain.
